util.py

```diff
@@ -0,0 +1,128 @@
+def process_seq(seq: str):
+    idx_low = [idx for idx, ch in enumerate(seq) if ch.islower()]
+    seq = seq.upper()
+    return idx_low, seq
+
+
+def output_seq(idx_low, seq: str):
+    seq = [ch for ch in seq]
+    for idx in idx_low:
+        seq[idx] = seq[idx].lower()
+    seq = "".join(seq)
+    return seq
+
+
+def check_acid(target: set, orig: set):
+    default = True
+    for i in target:
+        if i in orig:
+            continue
+        else:
+            default = False
+            break
+    return default
+
+
+def transfer(seq: str, dna=True):
+    seq = [ch for ch in seq]
+    if dna:
+        dict_sub = {"A": "T", "T": "A", "G": "C", "C": "G"}
+    else:
+        dict_sub = {"A": "U", "U": "A", "G": "C", "C": "G"}
+    for i in range(len(seq)):
+        seq[i] = dict_sub[seq[i]]
+    seq = "".join(seq)
+    return seq
+
+
+def transribe(seq: str):
+    idx_low, seq = process_seq(seq)
+    dna_alpha = {"A", "T", "G", "C"}
+    cur_seq = set(seq)
+    val = check_acid(cur_seq, dna_alpha)
+    if val:
+        seq = transfer(seq)
+        seq = output_seq(idx_low, seq)
+        return seq
+    else:
+        return "Invalid alphabet. Try again!"
+
+
+def reverse(seq: str):
+    rna_alpha = {"A", "U", "G", "C", "a", "u", "g", "c"}
+    dna_alpha = {"A", "T", "G", "C", "a", "t", "g", "c"}
+    cur_seq = set(seq)
+    val1 = check_acid(cur_seq, rna_alpha)
+    val2 = check_acid(cur_seq, dna_alpha)
```

Comment:
Странные имена для переменных
```suggestion
    is_rna = check_acid(cur_seq, rna_alpha)
    is_dna = check_acid(cur_seq, dna_alpha)
```